packages/agents-copilotstudio-client/src/copilotStudioClient.ts

```diff
@@ -206,7 +216,7 @@ export class CopilotStudioClient {
       responseType: 'stream',
       adapter: 'fetch'
     }
-    logger.info(`Asking question: ${question} ...`)
+    logger.info(`Sending activity: ${JSON.stringify(activity)} ...`)
```

Comment:
No need to do stringify, the logger can handle that.
```suggestion
    logger.info(`Sending activity...`, activity)
```

Reply:
Applied. Thanks!